src/game_api/script/usertypes/level_lua.cpp

```diff
@@ -1807,5 +1807,16 @@ void register_usertypes(sol::state& lua)
     // DEFAULT
     // FLOOR | SAFE | EMPTY
     */
+
+    lua.new_usertype<SpawnInfo>(
+        "SpawnInfo",
+        "room_temp",
```

Comment:
Probably shouldn't expose as "room_temp", it's ambiguous and called "room_template" everywhere else.